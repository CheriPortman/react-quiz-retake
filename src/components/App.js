import React from 'react';

export default function App() {
  return <h1>You can do this.</h1>;
}
