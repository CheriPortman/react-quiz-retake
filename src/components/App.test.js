import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Componenet', () => {
  it('renders the App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot(); 
  });
});
