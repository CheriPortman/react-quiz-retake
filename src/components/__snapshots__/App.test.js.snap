// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Componenet renders the App 1`] = `
<Fragment>
  <h1>
    You can do this.
  </h1>
  <Meme />
</Fragment>
`;
